components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -526,10 +526,14 @@ protected void buildClaimMappings(Map<ClaimMapping, String> claims, Map.Entry<St
 
             }
         } catch (Exception e) {
-            claimValue = new StringBuilder(entry.getValue().toString());
+            if (log.isDebugEnabled()) {
+                log.debug("Error while mapping " + entry.getKey() + " with value " + entry.getValue(), e);
```

Comment:
Could you please add a delimiter among the text and the value as well, as that adheres with the debug messaging practice being followed.
` log.debug("Error while mapping : " + entry.getKey() + " with value : " + entry.getValue(), e);`